src/routes/(app)/order/[id]/+page.svelte

```diff
@@ -508,6 +511,46 @@
 					{/each}
 				</ul>
 			{/if}
+			{#if data.order.status === 'pending' && remainingAmount && (data.roleId === CUSTOMER_ROLE_ID || !data.roleId)}
+				<form action="/order/{data.order._id}?/addPayment" method="post" class="contents">
+					<div class="flex flex-wrap gap-2">
+						<label class="form-label">
+							{t('order.addPayment.amount')}
+							<input
+								class="form-input"
+								type="number"
+								name="amount"
+								min="0"
+								step="any"
+								max={remainingAmount}
+								value={remainingAmount}
+								readonly
+							/>
+						</label>
+						<label class="form-label">
+							{t('order.addPayment.currency')}
+							<select name="currency" class="form-input" disabled>
+								<option value={data.order.currencySnapshot.main.totalPrice.currency}
+									>{data.order.currencySnapshot.main.totalPrice.currency}</option
+								>
+							</select>
+						</label>
+						<label class="form-label">
+							<span>{t('checkout.payment.method')}</span>
+							<select name="method" class="form-input">
+								{#each data.paymentMethods.filter( (pm) => ['card', 'bitcoin', 'lightning', 'paypal'].includes(pm) ) as paymentMethod}
```

Comment:
This should be computed up top in the `script` tag. Because we don't want to show the help text or this block if the payment method list is empty